dashboard/app/helpers/levels_helper.rb

```diff
@@ -160,6 +160,13 @@ def app_options
       view_options(authored_hint_view_requests_url: authored_hint_view_requests_path(format: :json))
     end
 
+    if @user
+      recent_driver_ul = UserLevel.find_by(script: @script, level: @level, user: @user).driver_user_levels.last
+      if recent_driver_ul
+        level_view_options pairing_driver: recent_driver_ul.user.name
```

Comment:
this appears to be mostly duplicated in [api_controller](#diff-d0ba960dee0b3ab3f04aebc9fb0e01c6R126)